docs/operatorTestPlaybooksCI.md

```diff
@@ -0,0 +1,52 @@
+# Purpose
+This document is to describe about the working of the Upstream Operator Test Playbooks CI Pipeline.
+
+# Introduction
+The Operator Test Playbooks CI Pipeline exists to automatically run tests against pull requests in the redhat-operator-ecosystem/operator-test-playbooks GitHub repository.
+
+# Configuration
+A webhook is configured so that the repository sends events to the github2fedmsg web application.
+Any activity (ex: adding comments) in the repository is therefore sent to github2fedmsg, which then rebroadcasts on the Fedora Messaging bus (fedmsg). 
+
+# Usage
+* The [test-operator-test-playbooks](https://jenkins-cvp-ci.cloud.paas.psi.redhat.com/job/test-operator-test-playbooks/) job runs on the CVP CI Jenkins server.
+
+* The job is triggered by the Fedora Messaging messages which:
+
+  * Indicate that a pull request was opened, re-opened, or synchronized (“synchronized” meaning that code changes were pushed to an already-opened pull request)
+
+    Ex: [PR-opened](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.pull_request.opened)
+
+  * Indicate a comment was added that contains the text `/retest` or `/test CVP`
+
+    Ex: [PR-comment-added](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.issue.comment&delta=3600)
+
+* The openshift project for the `test-operator-test-playbooks` is `cvp-ci-operatortestplaybooks`
```

Comment:
You can remove the second `the`: `The openshift project for test-operator-test-playbooks...`